central/complianceoperator/v2/checkresults/datastore/datastore.go

```diff
@@ -20,9 +21,12 @@ type DataStore interface {
 
 	// SearchCheckResults retrieves the scan results specified by query
 	SearchCheckResults(ctx context.Context, query *v1.Query) ([]*storage.ComplianceOperatorCheckResultV2, error)
+
+	// CheckResultStats retrieves the scan results stats specified by query
+	CheckResultStats(ctx context.Context, query *v1.Query) ([]*ResourceCountByResultByCluster, error)
```

Comment:
It happened again, I interpreted the `Check` keyword as examine.

Reply:
😆 I will try to be more careful and stop starting things with `check`.